File: NewYorkRangers/app/src/main/java/ru/sport/newyorkrangers/api/ApiService.java
package ru.sport.newyorkrangers.api;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;
import ru.sport.newyorkrangers.calendar.CalendarData;
import ru.sport.newyorkrangers.news.NewsData;
import ru.sport.newyorkrangers.statistics.StatisticsData;

public interface ApiService {
    @GET("news.json")
    Call<List<NewsData>> getNews();

    @GET("match.json")
    Call<List<CalendarData>> getCalendar();

    @GET("stats.json")
    Call<List<StatisticsData>> getStatistics();
}

File: KtSportApp/app/src/main/java/com/sport/ktsportapp/ApiService.kt
package com.sport.ktsportapp

import retrofit2.Response
import retrofit2.http.GET

interface ApiService {

    @GET("news.json")
    suspend fun getNews(): Response<List<News>>

    @GET("stats.json")
    suspend fun getHockeyStats(): Response<List<HockeyStat>>
}
